src/api/config/routes.rb

```diff
@@ -792,6 +792,9 @@ def self.public_or_about_path?(request)
       scope :published do
         get ':project_name/:repository_name/reports/:report_uuid' => :show, constraints: cons
       end
+      scope :built do
```

Comment:
I guess this should be :build - but it's interestingly relevant to all other code :)

Reply:
aehm, not relevant